projects/schematics/.eslintrc.json

```diff
@@ -1,4 +1,12 @@
 {
   "extends": "../../.eslintrc.json",
-  "ignorePatterns": ["**/*.d.ts"]
+  "ignorePatterns": ["**/*.d.ts"],
+  "overrides": [
+    {
+      "files": ["*.ts"],
+      "rules": {
+        "no-console": "off"
+      }
+    }
+  ]
```

Comment:
[QUESTION]

Why don't we need a similar config in:
- individual libraries' schematics
- e2e project folder
- tooling/scripts (e.g. `tools/schematics/testing.ts`)

?